features/partners.feature

```diff
@@ -153,378 +256,229 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
+      | burgerservicenummer | 999991553                                            |
+      | fields              | partners.burgerservicenummer,partners.naam.voornamen |
+      Dan heeft de persoon met burgerservicenummer '999991553' alleen 'partners' met de volgende gegevens
+      | burgerservicenummer |
+      | 555550004           |
+      En heeft de partner met burgerservicenummer '555550004' de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Karel  |
+  
+
+  @proxy
+  Rule: Wanneer de geslachtsnaam van de partner onbekend is, wordt de partner geleverd met type "OnbekendPartner" en indicatieOnbekend met waarde true
+    - Dit is het geval wanneer geslachtsnaam dan de standaardwaarde "." heeft
+    - Wanneer van de partner en het huwelijk/partnerschap wel gegevens geregistreerd zijn, maar geen van de met fields gevraagde gegevens heeft een waarde, dan is het type "Partner" en wordt indicatieOnbekend NIET opgenomen
+
+    # Onderliggende aanname is dat wanneer de geslachtsnaam van de partner onbekend is, ook andere (identificerende) partnergegevens niet bekend zijn of niet relevant.
 
     @proxy
-    Scenario: Niet leveren van partnergegevens waar niet om gevraagd is met fields
+    Scenario: Partner is volledig onbekend
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                | waarde    |
-      | burgerservicenummer | 999992807 |
-      En de persoon heeft de volgende naam gegevens
-      | naam                           | waarde |
-      | aanduiding naamgebruik (61.10) | P      |
-      En de persoon heeft exact 1 'partners'
-      En heeft de partner de volgende gegevens
+      | burgerservicenummer | 999993008 |
+      En de persoon heeft een partner met de volgende gegevens
       | naam                | waarde    |
-      | burgerservicenummer | 999992808 |
-      En heeft de partner de volgende 'geslachtsaanduiding' gegevens
-      | naam         | waarde |
-      | code         | M      |
-      | omschrijving | man    |
-      En de partner heeft de volgende naam gegevens
+      | burgerservicenummer |           |
+      En de partner heeft GEEN 'geslachtsaanduiding' gegevens
+      En de partner heeft alleen de volgende 'naam' gegevens
       | naam                                 | waarde    |
-      | voornamen                            | Cees      |
-      | voorvoegsel                          | de        |
-      | geslachtsnaam                        | Vries     |
-      En de partner heeft GEEN ontbindingHuwelijkPartnerschap gegevens
+      | geslachtsnaam                        | .         |
+      En de partner heeft alleen de volgende 'geboorte' gegevens
+      | naam                | waarde    |
+      | datum               | 00000000  |
+      En de partner heeft de volgende 'soortVerbintenis' gegevens
+      | naam         | waarde   |
+      | code         | .        |
```

Comment:
Hieruit blijkt dat de onbekende soortVerbintenis wel geleverd wordt. 
In het scenario "    @gba  Partner is volledig onbekend (dus niet onjuist) "
wordt de soortVerbintenis met een "." niet geretourneerd. 

Één van de twee moet aangepast worden.